apps/prairielearn/assets/stylesheets/pageLayout.css

```diff
@@ -360,3 +363,8 @@ body:has(div.app-main-container) {
     transition: none;
   }
 }
+
+.page-footer-text {
+  /* https://github.com/PrairieLearn/marketing/blob/6dc08c6028c8a45e71fce846b308326469958a16/src/components/Header.module.scss#L4-L6 */
+  color: #052c65; /* var(--bs-blue-800) */
```

Comment:
Why `var(--bs-blue-800)`? To me that would imply that `--bs-blue-800` is a CSS variable, but it's not.

I would just as soon make all this text gray like on the marketing site 😅

Reply:
I would prefer to change that in another PR, I like the current color.